<launch>

    <rosparam param="/gaussian_process/global_goal" value="0.05"/>

    <rosparam param="processing_frame" value="left_hand_palm_link"/>

	<include file="$(find pacman_vision)/launch/pacman_vision.launch">
		<arg name="separate_term" value="false"/>
		<arg name="run_rviz" value="false"/>
		<arg name="load" value="DR54"/>
		<arg name="launch_openni2" value="false"/>
	</include>

	<include file="$(find gp_regression)/launch/gp_node.launch">
		<arg name="run_rviz" value="false"/>
		<arg name="simulate" value="false"/>
	</include>

</launch>